R/preprocess.R

```diff
@@ -690,9 +709,8 @@ enw_construct_data <- function(obs, new_confirm, latest, missing_reference,
 #' that this is zero indexed and so includes the reference date and
 #' `max_delay - 1` other days.
 #'
-#' @param holidays A vector of dates indicating when holidays occur used by
-#' [enw_add_metaobs_features()] to treat holidays as sundays within the
-#' `day_of_week` variable it creates internally.
+#' @param ... other arguments to [enw_add_metaobs_features()],
```

Comment:
start with caps

Reply:
(perhaps explain very briefly what this function is and why you might want to look.